python/pyspark/context.py

```diff
@@ -22,22 +22,30 @@
 import signal
 import sys
 import threading
-from threading import RLock
 from tempfile import NamedTemporaryFile
+from threading import RLock
 
 from pyspark import accumulators
 from pyspark.accumulators import Accumulator
 from pyspark.broadcast import Broadcast
 from pyspark.conf import SparkConf
 from pyspark.files import SparkFiles
 from pyspark.java_gateway import launch_gateway
-from pyspark.serializers import PickleSerializer, BatchedSerializer, UTF8Deserializer, \
-    PairDeserializer, AutoBatchedSerializer, NoOpSerializer
-from pyspark.storagelevel import StorageLevel
-from pyspark.rdd import RDD, _load_from_socket, ignore_unicode_prefix
-from pyspark.traceback_utils import CallSite, first_spark_call
+from pyspark.profiler import BasicProfiler
+from pyspark.profiler import ProfilerCollector
+from pyspark.rdd import RDD
+from pyspark.rdd import _load_from_socket
+from pyspark.rdd import ignore_unicode_prefix
+from pyspark.serializers import AutoBatchedSerializer
```

Comment:
indeed this is a deviation from Pep recommendation. I encourage this behavior since it simplifies a lot file maintainance.

On our Buildbot based project, we use to have lot of conflict involving changes on import statement: on 2 differents branches (say: prod and main), we often had to import either the same import (so merge might add this line twice when the two developers have placed them in two different places) or not so easy to solve conflict (when two developers add theo import two different object from the same module).
Once we setup the "one import per line" rule, no more conflict on this lines, ever. This helped us a lot automatizing an auto merger from the release branches to the "master" branch.